ramls/actual-cost-record.json

```diff
@@ -96,6 +96,10 @@
       "description": "Permanent item location of the lost item",
       "type": "string"
     },
+    "accountId": {
+      "description": "Account ID of the fee",
```

Comment:
ID of the fee/fine created based on the actual cost record during its manual processing